apps/gro/controllers/address.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const BaseController = require('hof').controllers.base;
+const ErrorController = require('hof').controllers.error;
+const _ = require('lodash');
+
+module.exports = class Address extends BaseController {
+  constructor(options) {
+    super(options);
+  }
+
+  getValues(req, res, callback) {
+    res.locals.backLink = req.sessionModel.get('previous-step');
```

Comment:
isn't the `backLink` set to `locals` anyway, I thought you got this for free, do you need to set it manually?